app/components/offers-block.js

```diff
@@ -0,0 +1,40 @@
+import { inject as service } from "@ember/service";
+import { getWithDefault } from "@ember/object";
+import Component from "@ember/component";
+import _ from "lodash";
+
+export default Component.extend({
+  store: service(),
+  locationName: "",
+
+  didReceiveAttrs() {
+    this._super(...arguments);
+    console.log(this.get("record"));
+    const location = this.get("store").peekRecord(
+      "district",
+      this.get("record").district_id
+    );
+    this.getPackageTypesCounts(this.get("record").items);
+    this.set("locationName", getWithDefault(location, "name", "N/A"));
+    this.set("initialItems", this.get("record").items.slice(0, 4));
+  },
+
+  getPackageTypesCounts(items) {
```

Comment:
```suggestion
  computePackageTypesCounts(items) {
```

Functions starting with "get" (`getXXX`) usually mean that they return something. This one doesn't return anything, it just sets stuff. Thus my renaming